File: springboot/open-api/src/main/java/com/openapi/domain/dto/ws/request/ControlCommandRequest.java
package com.openapi.domain.dto.ws.request;

import com.openapi.domain.dto.ws.base.CommonResultDto;
import lombok.Data;
import lombok.EqualsAndHashCode;
import java.util.Map;

@EqualsAndHashCode(callSuper = true)
@Data
public class ControlCommandRequest extends CommonResultDto {
    private String deviceId;
    private String command;
    private Map<String, String> params;
}

File: project/app/android/app/src/main/java/com/magicvector/domain/dto/ws/request/ControlCommandRequest.kt
package com.magicvector.domain.dto.ws.request

import com.magicvector.domain.dto.ws.base.CommonResultDto
import java.util.*

data class ControlCommandRequest(
    val deviceId: String,
    val command: String,
    val params: Map<String, String>
) : CommonResultDto()